scripts/api/data/licenses/licenseOGL-UK-3.0.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "OGL UK 3.0",
+  "uri": "https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3",
```

Comment:
This seems to be the right uri. There's only one: https://github.com/datacite/bracco/blob/df4ac2030774324adeaa72e2978c29dfb463ee3d/app/spdx.js#L3151